wagtail/snippets/templates/wagtailsnippets/snippets/action_menu/workflow_menu_item.html

```diff
@@ -0,0 +1,4 @@
+{% load wagtailadmin_tags %}
```

Comment:
Why can't this be shared?

Reply:
As noted in the comment above, we're currently not sharing any of the action menu items, and I think it's best to unify them both together later.